test/dotnet.Tests/GivenThatTheUserIsRunningDotNetForTheFirstTime.cs

```diff
@@ -28,6 +28,7 @@ static GivenThatTheUserIsRunningDotNetForTheFirstTime()
                 .WithWorkingDirectory(testDirectory.Path);
             command.Environment["NUGET_PACKAGES"] = testNugetCache;
             command.Environment["DOTNET_SKIP_FIRST_TIME_EXPERIENCE"] = "";
+            command.Environment["SkipInvalidConfigurations"] = "true";
```

Comment:
This shouldn't be necessary since this command only call `--info` and `new`.  It doesn't do anything with MSBuild.